easybuild/easyconfigs/s/SHAPEIT4/SHAPEIT4_4.2.0_use_EB_settings.patch

```diff
@@ -0,0 +1,38 @@
+--- shapeit4-4.2.0/makefile	2021-01-21 08:05:26.000000000 +0000
```

Comment:
Please add a description and author information to the patch.

Reply:
Done. It's essentially just a re-jigging of the previous patch.